utils/src/AzExtTreeFileSystem.ts

```diff
@@ -70,7 +69,7 @@ export abstract class AzExtTreeFileSystem<TItem extends AzExtTreeItem> implement
         await callWithTelemetryAndErrorHandling('writeFile', async (context) => {
             const item: TItem = await this.lookup(context, uri);
             await this.writeFileImpl(context, item, content, uri);
-            await item.refresh(context);
+            // await item.refresh(context);
```

Comment:
I will have to do more investigation on what this was doing and if we need to replicate this behavior still.